testGradient.m

```diff
@@ -55,7 +55,17 @@
 end
 
 if nargin >= 6
-    fplot = varargin{6};
+    if(~isempty(varargin{6}))
+        plist = varargin{6};
+    else
+        plist = 1:length(theta);
+    end
+else

```

Comment:
It's not a big thing, but could we please use the same indentation style for the main function of test gradient as for the subfunctions?
Thanks.

Reply:
Related: #50

Reply:
done